### ISSUE TITLE
Javascript error prevents some server consoles from working

### ISSUE THREAD

**Issue by BrainStone** [Pterodactyl/Panel#686]
I'm experiencing a strange behavior with the server consoles. When I open the server page where the console is at it stays blank for some servers. On these servers it non operational and even the Start/Stop/... buttons don't work. Upon closer inspection I noticed that the following error appears in the browser console: `TypeError: data.line is undefined  console.js:200:9`  
However this error only appears on servers with a certain service. Namely the Waterfall service (Waterfall is an alternative BungeeCord) I created myself. It used to work before though. And since I haven't used the panel in a good month I can't exactly pinpoint when it stopped working. I also had a hard shutdown on the server during the period of no usage.  
It is also very weird that when I start the server the console works while I stay on the page. As soons as I leave and come back to it it stops working.

I'll happily provide additional information when requested.

I'm using the panel version 0.6.4 and do not use any themes. Other versions below:

`uname -a`:
```
Linux BrainHost 4.12.0-1-amd64 #1 SMP Debian 4.12.6-1 (2017-08-12) x86_64 GNU/Linux
```
`php -v`:
```
PHP 7.1.8-1 (cli) (built: Aug  3 2017 18:35:23) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.1.0, Copyright (c) 1998-2017 Zend Technologies
    with Zend OPcache v7.1.8-1, Copyright (c) 1999-2017, by Zend Technologies
```
`docker info`:
```
Containers: 3
 Running: 3
 Paused: 0
 Stopped: 0
Images: 10
Server Version: 17.09.0-ce
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file logentries splunk syslog
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 06b9cb35161009dcb7123345749fef02f7cea8e0
runc version: 3f2f8b84a77f73d38244dd690525642a72156c64
init version: 949e6fa
Security Options:
 seccomp
  Profile: default
Kernel Version: 4.12.0-1-amd64
Operating System: Debian GNU/Linux 9 (stretch)
OSType: linux
Architecture: x86_64
CPUs: 8
Total Memory: 31.34GiB
Name: BrainHost
ID: BX5U:Q3RA:MQL6:GFJ5:PKCG:7YXX:APWL:DCLP:BMQW:OSRZ:ZSF3:54GA
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false

WARNING: No swap limit support
```

**Comment by BrainStone** [comment]
According to https://github.com/Pterodactyl/Panel/blame/develop/public/themes/pterodactyl/js/frontend/console.js#L200 @DaneEveritt last changed that line.

**Comment by ShadowKitten** [comment]
Definitely having this issue too it's quite annoying.

**Comment by ShadowKitten** [comment]
After wrapping data.line with String() at https://github.com/Pterodactyl/Panel/blame/develop/public/themes/pterodactyl/js/frontend/console.js#L200 I was able to get everything working again but at the beginning of the console I see "undefined" what this means to me is that the initial log is not being loaded.

Recently waterfall a fork of Bungeecord made some changes to the way they handle and store log files. This is likely incompatible with the service configuration included with Pterodactyl, though it still shouldn't be crashing the console just because there is a missing log file.

**Comment by BrainStone** [comment]
Ok. After correcting the log path it is working again.

So I guess the base issue is that if the configured log file can't be found that the error appears. It should be handled and if possible an error message indicating that the log could not be found should appear.

**Comment by adamxp12** [comment]
Fixed with #710
